package.json

```diff
@@ -4,29 +4,33 @@
   "devDependencies": {
     "@types/jasmine": "^3.5.10",
     "clang-format": "^1.2.2",
+    "generate-license-file": "^1.2.0",
     "husky": "^1.3.1",
     "jasmine": "^3.5.0",
+    "prettier": "^2.4.1",
+    "pretty-quick": "^3.1.1",
     "tslint": "^5.9.1",
     "typescript": "^4"
   },
   "engines": {
-    "yarn": ">=1.0",
-    "node": "^12"
+    "node": ">=16.13.0",
```

Comment:
Are you sure you want ">="?  I believe this auto-upgrades across major versions, which are not required to be backward-compatible.

Reply:
Mm, this was an artifact of trying to parse this, i didn't want to have to deal with the full syntax 😅  will add `<17`/`<9` to the end

Reply:
It sounds like we don't need to parse this any more, so we can use proper semantic versioning.